packages/@aws-cdk/aws-lambda/test/runtime-management.test.ts

```diff
@@ -0,0 +1,67 @@
+import { Template } from '@aws-cdk/assertions';
+import * as cdk from '@aws-cdk/core';
+import * as lambda from '../lib';
+
+describe('runtime', () => {
+  test('Runtime Management Auto', () => {
+    // GIVEN
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, 'stack');
+    new lambda.Function(stack, 'Lambda', {
+      code: new lambda.InlineCode('foo'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_18_X,
+      runtimeManagement: {
+        mode: lambda.UpdateRuntimeOn.AUTO,
+      },
+    });
+    // WHEN
+    Template.fromStack(stack).hasResourceProperties('AWS::Lambda::Function', {
+      RuntimeManagementConfig: {
+        UpdateRuntimeOn: 'Auto',
+      },
+    });
+  });
+  test('Runtime Management Function Update', () => {
+    // GIVEN
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, 'stack');
+    new lambda.Function(stack, 'Lambda', {
+      code: new lambda.InlineCode('foo'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_18_X,
+      runtimeManagement: {
+        mode: lambda.UpdateRuntimeOn.FUNCTION_UPDATE,
+      },
+    });
+    // WHEN
+    Template.fromStack(stack).hasResourceProperties('AWS::Lambda::Function', {
+      RuntimeManagementConfig: {
+        UpdateRuntimeOn: 'Function update',
+      },
+    });
+  });
+  test('Runtime Management MANUAL', () => {
```

Comment:
we need a test to verify that the `manual` case throws an error if the `arn` is unspecified